source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md

```diff
@@ -81,6 +81,34 @@ These methods are:
 * Checkout actions:
     - `finish`
 
+### Variants in listing
+
+It is now possible to allow for filtering and display of variants in the listing. This new filter can be activated in the
```

Comment:
It is now possible to allow displaying and filtering for variants in the listing.